models/apartment.rb

```diff
@@ -0,0 +1,6 @@
+class Apartment < ActiveRecord::Base
+
+has_many :tenants
+belongs_to :tenants
```

Comment:
Should an apartment belong to a tenant? A one-to-many relationship between apartments and tenants works fine for this assignment.

Also, `belongs_to` should accept a singular argument. It doesn't apply here, but if you were to use it for these models you would write `belongs_to :tenant`.